.github/workflows/npmPublish.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```

Comment:
This indicates that the job will run on every release. We want the job to be strictly manual for now.

Reply:
added manual configuration on npm publish